frontend/talentsearch/src/js/components/languageInformationForm/ConsideredLanguages.tsx

```diff
@@ -282,8 +282,8 @@ const ConsideredLanguages = ({ labels }: ConsideredLanguagesProps) => {
             {intl.formatMessage(
               {
                 defaultMessage:
-                  "If you want to find out your language proficiency levels, <selfAssessmentLink>click here to find out.</selfAssessmentLink>",
-                id: "nVh2Qh",
+                  "You can find out find out your levels with a <selfAssessmentLink>language proficiency self assessment</selfAssessmentLink>.",
```

Comment:
**self assessment** to **self-assessment**. this will also change the id but this should not change the french string's `defaultMessage` (though you could send it again and see if it does to be sure as i have seen both [auto-évaluation](https://www.canada.ca/fr/commission-fonction-publique/services/outils-ressources-dotation-evaluation/specialistes-ressources-humaines-gestionnaires-embaucheurs/boite-outils-gestion-rh/securite-examens/auto-evaluation-liste-verification-erreurs-matiere-securite-administration-examens.html) and [autoévaluation](https://www.canada.ca/fr/commission-fonction-publique/services/evaluation-langue-seconde/tests-autoevaluation.html) and the opposite in English (to make this evermore confusing) on GC websites). 

```suggestion
                  "You can find out find out your levels with a <selfAssessmentLink>language proficiency self-assessment</selfAssessmentLink>.",
```

Reply:
I think we can use to non hyphenated version to match the site we are linking to as you noted.

Reply:
The site we are linking to uses the hyphenated version in English and non-hyphenated in French.

![Screen Shot 2022-11-10 at 12 24 02](https://user-images.githubusercontent.com/3046459/201170633-23a77522-4287-4434-9330-9c7466c95056.png)

Reply:
oh yeah, sorry. I was specifically talking about French 😅